frontend/tailwind.config.ts

```diff
@@ -98,6 +98,12 @@ export default {
       pink: shadeScale("pink"),
       yellow: shadeScale("yellow"),
 
+      // Focus ring colors
+      focus: {
+        DEFAULT: "var(--color-border-focus)",
+        yellow: "var(--color-yellow-3",
```

Comment:
```suggestion
        yellow: "var(--color-yellow-3)",
```

Reply:
How did it work for me locally 🤔

Reply:
CSS is surprisingly flexible when it wants to be :grin: